### PR TITLE
[IMP] mass_mailing: add basic image shapes edi

### PR DESCRIPTION
**Pull request by edi-odoo**
Since the Style option on images contains unsupported styles on images -
shadows aren't supported, the rounded rectangle (which is not supported
in Outlook) can be replaced by the border's options which also allow a
more customized border-radius, the thumbnail styling on images can be
done with the border and padding options - it has been removed from
images and replaced with a circle shape and other options mentioned
above. However, these options remain available on icons.

This commit adds three basic shapes (a circle, a triangle and a
"slanted" rectangle) to the web_editor's "Basics" shapes section.
This section is disabled because of a condition set on the
section; the condition is moved to each of its shapes instead, so that
new shapes from mass_mailing can be added via xpath into this section.

In `theme_tech` in `mass_mailing_themes`, the static triangular image 
has now been switched to a shaped image.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web_editor/views/snippets.xml [hunk 1]
```diff
@@ -448,18 +448,18 @@
...
-                        <we-button data-set-img-shape="web_editor/basic/bsc_square_5" data-select-label="Square 5" data-animated="true"/>
-                        <we-button data-set-img-shape="web_editor/basic/bsc_square_6" data-select-label="Square 6" data-animated="true"/>
-                        <we-button data-set-img-shape="web_editor/basic/bsc_square_rounded_1" data-select-label="Square Rounded 1" data-animated="true"/>
-                        <we-button data-set-img-shape="web_editor/basic/bsc_square_rounded_2" data-select-label="Square Rounded 2" data-animated="true"/>
-                        <we-button data-set-img-shape="web_editor/basic/bsc_organic_1" data-select-label="Organic Soft" data-animated="true"/>
-                        <we-button data-set-img-shape="web_editor/basic/bsc_organic_2" data-select-label="Organic Medium" data-animated="true"/>
-                        <we-button data-set-img-shape="web_editor/basic/bsc_organic_3" data-select-label="Organic Hard" data-animated="true"/>
+                    <we-select-page class="o_we_basic_shapes" string="Basics">
```
Comment (Zinston) [inline addons/web_editor/views/snippets.xml:451]: Why did you move the `t-if` to the individual buttons? IIRC it was on the parent because otherwise we could end up with an empty page, since we have to hide the _whole page_ if we don't allow animations.

**Review by Zynton — CHANGES_REQUESTED** [review]
Nice! Only one comment on the code, and also I notice many of your commits don't include a description, and none include the task number. Could you please fix that, using the [guidelines](https://www.odoo.com/documentation/15.0/contributing/development/coding_guidelines.html#commit-message-structure)? Thanks!

File: addons/web_editor/views/snippets.xml (hunk 1 above)
Comment (edi-odoo) [reply addons/web_editor/views/snippets.xml:451]: Indeed, we don't want an empty page and that's why it was on the whole page. But I added 3 shapes in Basics that we *do* want to show in mass_mailing. (if we hide the whole page, we can't see the three basic shapes I added). 
see https://github.com/odoo/odoo/pull/95974/files/81eb4cf0eae7bf9d8796a90f29a9a8001b45dc5b#diff-b26221e9411fb6f3725ce5a30c4659a22d81a9a69f666bdbd6daa77e03ec3253R350

Comment (Zinston) [reply addons/web_editor/views/snippets.xml:451]: I see. Does it work to remove the t-if on `we-select-page` with your xpath? So that we don't end up with an empty page in the contexts where your xpath is not active?

I mean you'd leave the t-ifs you added on each button, put back the t-if on we-select-page, and then in your xpath remove that t-if from we-select-page. I don't know in which order things are done and so I'm not sure you can remove a t-if that way but I think it's worth a shot. @dmo-odoo What do you think?

**Review by Zynton — APPROVED** [review]

**Comment by Zinston** [comment]
@robodoo rebase-ff r+

**Comment by dmo-odoo** [comment]
@robodoo r+
